docs/api.md

```diff
@@ -303,6 +304,16 @@ for `code` and `errno` are:
   Redis WATCH detected a conflicting update
 * `code: 400, errno: 166`:
   Not a public client
+* `code: 400, errno: 167`:
```

Comment:
We'll need all of these new errors surfaced in the content-server's [auth-errors.js list](https://github.com/shane-tomlinson/fxa-content-server/blob/3803c6ef1e6787aea963e37e99b1281aa2c3eebb/app/scripts/lib/auth-errors.js#L238).

Reply:
IIUC this could be done as part of the future content-server PR that uses this endpoint in the oauth flow.

Reply:
https://github.com/mozilla/fxa-content-server/issues/7098